cou/cli.py

```diff
@@ -92,18 +96,29 @@ async def analyze_and_plan(model_name: Optional[str] = None) -> tuple[Analysis,
     :return: Generated analyses and upgrade plan.
     :rtype: tuple[Analysis, UpgradeStep]
     """
+    progress_indicator.start(f"Connecting to '{model_name or 'default'}' model...")
```

Comment:
Also since you are outputting the model_name here, it's good to close issue #127 with this PR

Reply:
You are right I should use `current-model` and not `default`. Will change it and I add #127.

Reply:
As discussed in sync, to avoid accessing juju credential files to find the current model name before establishing the connection, we will simply use "current model" in the `Connecting ...` message if no model name is provided by user. And then in progress_indicator.success(), we supply the actual model name as part of the persisted text

Reply:
As this is now being handled by https://github.com/canonical/charmed-openstack-upgrader/pull/171, resolving the conversation here